build.gradle.kts

```diff
@@ -488,46 +520,19 @@ tasks {
     }
   }
 
-  patchPluginXml {
-    sinceBuild = properties("pluginSinceBuild")
-    untilBuild = properties("pluginUntilBuild")
-  }
+  val customRunIde by
```

Comment:
Do we need separate task?
If we can set version on `runIde` then we can do the same, just with fallback to `platformVersion` if `platformRuntimeVersion` is not available.

Reply:
sadly, it does not work yet